knock/src/core/ui/Toast/Toast.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import styles from '../../styles/toast.module.scss';
+
+const Toast: React.FC = () => {
+  const [alert, alertSet] = useState(true);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      alertSet(false);
+    }, 2000);
+
+    return () => clearTimeout(timer);
+  }, []);
+
```

Comment:
기능 부분을 상위에서 다루는게 더 편할 것 같은데 어떻게 생각하시나요?? 😄
```suggestion
interface ToastProps {
  토스트_나타내기_상태값: boolean;
  토스트_메세지: string;
}
const Toast: React.FC<ToastProps> = ({토스트_나타내기_상태값, 토스트_메세지}) => {
```